SecurityPkg/Tcg/Tcg2Smm/Tcg2Smm.c

```diff
@@ -86,6 +86,13 @@ TpmNvsCommunciate (
   switch (CommParams->Function) {
     case TpmNvsMmExchangeInfo:
       DEBUG ((DEBUG_VERBOSE, "[%a] - Function requested: MM_EXCHANGE_NVS_INFO\n", __FUNCTION__));
+      // MU_CHANGE TCBZ4378 [BEGIN] - Check for invalid NVS buffer location
+      if (IsBufferOutsideMmValid (CommParams->TargetAddress, sizeof (TCG_NVS))) {

```

Comment:
Was this intended to be `!IsBufferOutsideMmValid()`?

```suggestion
      if (!IsBufferOutsideMmValid (CommParams->TargetAddress, sizeof (TCG_NVS))) {
```

Reply:
Nice catch, thanks.